index.html

```diff
@@ -23,22 +23,147 @@ <h1>Hey There.i'm Said</h1>
+        </ul>
+        <form class="check">
+          <button type="button" class="click">See Project</button>
+        </form>
+      </section>
+      <section class="work project1">
+        <div class="linfo">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="lang">
+            <li class="ht">html</li>
+            <li class="bo">bootstrap</li>
+            <li class="ru">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="but">See Project</button>
+        </form>
+      </section>
+      <section class="work projectq">
+        <div class="linfoq">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="langq">
+            <li class="htq">html</li>
+            <li class="boq">bootstrap</li>
+            <li class="ruq">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="butq">See Project</button>
+        </form>
+      </section>
+      <section class="work projectw">
+        <div class="linfow">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="langw">
+            <li class="htw">html</li>
+            <li class="bow">bootstrap</li>
+            <li class="ruw">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="butw">See Project</button>
+        </form>
+      </section>
+      <section class="work projectX">
+        <div class="linfoe">
+          <h2>Profesional Art Printing Data</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+          <ul class="leng">
+            <li class="hte">html</li>
+            <li class="strap">bootstrap</li>
+            <li class="rube">Ruby</li>
+          </ul>
+        </div>
+        <form>
+          <button type="button" class="bute">See Project</button>
+        </form>
+      </section>
+      <section class="work about">
+        <h2>About me</h2>
+        <p class="par">Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
+        <form>
+          <button type="button" class="abouut">Get My Resume</button>
+        </form>
+        <div class="vic"></div>
+        <section class="langue">
+          <div class="Eng">
+            <h3 class="title0">Language</h3>
+            <section class="map">
```

Comment:
- [ ] You should use a `<div>` or `<article>` here for this is a sub section for semantic reasons.

Reply:
done